src/Npgsql.NetTopologySuite/PostGisGeometryType.cs

```diff
@@ -0,0 +1,45 @@
+namespace Npgsql.NetTopologySuite
+{
+    /// <summary>
+    /// PostGIS Geometry types as defined in the OGC WKB Spec
+    /// </summary>
+    internal enum PostGisGeometryType
```

Comment:
Do we write `PostGis` throughout? Or `PostGIS`?

Reply:
This is just a copy-paste of the NTS source - the idea is for this to disappear as soon as the fixes are merged upstream and released. So I definitely did not attempt to clean it up or anything (believe me it was hard to resist). FWIW I opened https://github.com/NetTopologySuite/NetTopologySuite.IO.PostGis/issues/6 to see if I can do some proper cleanup.